playground/package.json

```diff
@@ -1,17 +1,15 @@
 {
   "name": "playground",
   "version": "1.0.0",
-  "private": true,
   "scripts": {
     "demo": "nuxt",
     "build": "nuxt build",
     "start": "nuxt start",
     "generate": "nuxt generate"
   },
-  "dependencies": {
-    "@nuxtjs/composition-api": "^0.32.0",
+  "devDependencies": {
     "@storyblok/nuxt-2": "^0.0.1",
-    "core-js": "^3.22.2",
+    "core-js": "^3.15.1",
```

Comment:
Just curious, why did you have to downgrade this?

Reply:
Only life knows... fixing it ;)